public/app/core/services/backend_srv.ts

```diff
@@ -1,46 +1,56 @@
+// @ts-ignore
 import _ from 'lodash';
+// @ts-ignore
```

Comment:
why ts ignore on angular, we have types for angular included that seem to work, do you get an error for this line ?

Reply:
My bad, fixed it.